config/config-base.yml

```diff
@@ -34,6 +34,12 @@ run_name: ""
 # CRS EPSG code that project outputs should be in (projection should be in meter units and intended for the project area)
 project_crs: "EPSG::26910" # 26910 is UTM 10N
 
+# If True, add in a shift to the CRS so the mesh origin is at the camera EXIF average location and points are reported in a local frame - shifted from the normal CRS origin.
+# This is necessary when fine mesh detail is needed, since the natural export format of Metashape is float32 for meshes. In most CRS, that results in point quantization in the 5cm-20cm range
+# In order to save finer detail, set shift_crs_to_cameras to True, and then apply the inverse shift to the mesh model to recreate the true CRS values
+# The shift used is reported using the save_metadata_xml mechanism of exportModel
+shift_crs_to_cameras: False
```

Comment:
nit: I'd put this under the buildMesh heading now that it's only applicable to that and not the pointcloud. Unless there's a reason it should be project level?

Reply:
Done